tox.ini

```diff
@@ -4,18 +4,15 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27
```

Comment:
Instead of removing this entry, I think we need to update it. I believe this is what is executed when you run `tox` without manually specifying `-e <list of environments>`.